moveit_demo_nodes/run_moveit_cpp/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>run_moveit_cpp</name>
```

Comment:
this package naming is a break from our previous convention of always prepending with ``moveit_*``... it feels messy. Should be ``moveit_cpp_demo`` IMHO

Also needs a pkg description.

Reply:
Will address in the follow-up